excel/pom.xml

```diff
@@ -49,6 +49,25 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
       </plugin>
+      
+       <plugin>
+           <groupId>org.apache.maven.plugins</groupId>
+           <artifactId>maven-shade-plugin</artifactId>
+           <version>3.1.0</version>
```

Comment:
Can we move all the common configuration under `pluginManagement` in the parent pom?